runtime-sdk/src/modules/rofl/config.rs

```diff
@@ -16,6 +16,8 @@ pub trait Config: 'static {
     const GAS_COST_CALL_AUTHORIZED_ORIGIN_NODE: u64 = 2000;
     /// Gas cost of rofl.AuthorizedOriginEntity call.
     const GAS_COST_CALL_AUTHORIZED_ORIGIN_ENTITY: u64 = 2000;
+    /// Gas cost of rofl.StakeThresholds call.
+    const GAS_COST_CALL_STAKE_THRESHOLDS: u64 = 100;
```

Comment:
I have no idea what gas cost should we apply for fetching a constant from the config?

Reply:
Probably 20? See https://github.com/oasisprotocol/oasis-sdk/blob/main/runtime-sdk/src/modules/core/mod.rs#L426-L429

Why would one pay for the gas if the value could be hard-coded?

Reply:
The value can change. And yeah something small seems appropriate.

Reply:
Set to 10 to match `GAS_COST_CALL_CURRENT_EPOCH`.